[2022-11-27T10:01:46.106+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2022-11-27T10:01:33.178732+00:00 [queued]>
[2022-11-27T10:01:46.116+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2022-11-27T10:01:33.178732+00:00 [queued]>
[2022-11-27T10:01:46.117+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T10:01:46.118+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-27T10:01:46.119+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-27T10:01:46.131+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): store_user> on 2022-11-27 10:01:33.178732+00:00
[2022-11-27T10:01:46.141+0000] {standard_task_runner.py:55} INFO - Started process 5441 to run task
[2022-11-27T10:01:46.149+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'manual__2022-11-27T10:01:33.178732+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp0sikce9s']
[2022-11-27T10:01:46.154+0000] {standard_task_runner.py:83} INFO - Job 42: Subtask store_user
[2022-11-27T10:01:46.252+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.store_user manual__2022-11-27T10:01:33.178732+00:00 [running]> on host c7f7231a8603
[2022-11-27T10:01:46.314+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2022-11-27T10:01:33.178732+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-27T10:01:33.178732+00:00
[2022-11-27T10:01:46.319+0000] {postgres.py:129} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/process_user.csv
[2022-11-27T10:01:46.332+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2022-11-27T10:01:46.341+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-11-27T10:01:46.351+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20221127T100133, start_date=20221127T100146, end_date=20221127T100146
[2022-11-27T10:01:46.401+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-27T10:01:46.432+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
